sdk/create-backend.ts

```
@@ -0,0 +1,53 @@
+import { Function, type Api, type Bucket, type Stack } from 'sst/constructs'
```

Biome (1.9.4)

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)